study-builder/studies/brugada/snippets/picklist-0-to-4.conf

```diff
@@ -0,0 +1,64 @@
+{
+  "picklistOptions": [
+    {
+      "stableId": "0",
```

Comment:
Why are we having stable Ids 0 , 1 , 2, ... rather than a meaningful text ?
These might look odd in CSV export, elastic exports and DSM . 
Did you get a chance to test all of these and did data look meaningful in csv export and DSM ?

Reply:
Can we please change stable Id's from 1, 2, 3 to some meaningful strings that reflect what that option is about.

Reply:
@ssettipalli As I said in the comment above, it is exactly what it should be:
<img width="1471" alt="image" src="https://user-images.githubusercontent.com/71830706/135411943-c4d17545-dafb-4655-98ee-63b7ddb2cfb5.png">

Reply:
We could have named the options like "strongly disagree", ...., "strongly agree" but we don't know what's between these

Reply:
@ssettipalli I don't quite understand my next steps. What should I put there?